src/iidm/NetworkIndex.cpp

```diff
@@ -70,6 +98,21 @@ Identifiable& NetworkIndex::get(const std::string& id) {
     return const_cast<Identifiable&>(static_cast<const NetworkIndex*>(this)->get<Identifiable>(id));
 }
 
+const Identifiable& NetworkIndex::get(const std::string& idOrAlias) const {
+    auto aliasIt = m_idByAlias.find(idOrAlias);
+    const std::string& id = aliasIt != m_idByAlias.end() ? aliasIt->second : idOrAlias;
```

Comment:
You should create a utility function in `stdcxx/map.hpp` to simulate the getOrDefault java method